2024/opmodes/TwoStickTeleop.java

```diff
@@ -74,6 +75,11 @@ private enum Direction {
     private DcMotor hangingMotor;
     private DcMotor liftMotor;
 
+    private DcMotor intakeMotor;
+    private DcMotor transportMotor;
+//    private OneWheelDriveTask intakeMotorTask;
+//    private OneWheelDriveTask transportMotorTask;
```

Comment:
these commented out lines should be removed. If the plan is to go back to OneWheelDriveTask in the future, then that can be a new Github issue